### PR TITLE
Update sdk-amd64.wxs

### PR DESCRIPTION
**Pull request by compnerd**
Update installer manifest for new introduced dispatch header.

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @MaxDesiatov

**Review by MaxDesiatov — APPROVED** [review]
Thank you!

**Comment by compnerd** [comment]
Unfortunately, only @shahmishal can approve on this repo and that is required - please make sure that we get this in ASAP (assuming that apple/swift#63216 is good).  We should also retrigger a nightly snapshot build after this is merged - the Windows builds haven't been updated in a while.

File: platforms/Windows/sdk-x86.wxs [hunk 1]
```diff
@@ -211,6 +211,9 @@
         <File Id="time.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\time.h" Checksum="yes" />
       </Component>
 
+      <Component Id="generic_base.h" Directory="WindowsSDK_usr_include_dispatch" Guid="c3048f41-e208-45ff-a88f-682ff8292eb8">
+        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\group.h" Checksum="yes" />
```
Comment (MarSe32m) [inline platforms/Windows/sdk-x86.wxs:215]: Sorry for the random drive-by comment, but I think this ought to be
```suggestion
        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\generic_base.h" Checksum="yes" />
```

**Review by shahmishal — APPROVED** [review]

File: platforms/Windows/sdk-x86.wxs (hunk 1 above)
Comment (compnerd) [reply platforms/Windows/sdk-x86.wxs:215]: Close, but no dice, mostly because you didn't point out the other mistake in this 😂.  I messed up the location `os/generic_base.h`.  Thanks though!

**Comment by compnerd** [comment]
This will need to be squashed before merging.

File: platforms/Windows/sdk-arm64.wxs [hunk 2]
```diff
@@ -211,6 +211,9 @@
         <File Id="time.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\time.h" Checksum="yes" />
       </Component>
 
+      <Component Id="generic_base.h" Directory="WindowsSDK_usr_include_os" Guid="a136a05a-acc4-489b-96fc-1413f8421715">
+        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\os\generic_base.h" Checksum="yes" />
```
Comment (MaxDesiatov) [inline platforms/Windows/sdk-arm64.wxs:215]: `dispatch` component is not needed here, as it's a peer to `generic_unix_base.h` and `generic_win_base.h`, which are both in `usr\lib\swift\os`, not `usr\lib\swift\dispatch\os`
```suggestion
        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\os\generic_base.h" Checksum="yes" />
```

File: platforms/Windows/sdk-x86.wxs [hunk 3]
```diff
@@ -211,6 +211,9 @@
         <File Id="time.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\time.h" Checksum="yes" />
       </Component>
 
+      <Component Id="generic_base.h" Directory="WindowsSDK_usr_include_os" Guid="c3048f41-e208-45ff-a88f-682ff8292eb8">
+        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\os\generic_base.h" Checksum="yes" />
```
Comment (MaxDesiatov) [inline platforms/Windows/sdk-x86.wxs:215]:
```suggestion
        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\os\generic_base.h" Checksum="yes" />
```

File: platforms/Windows/sdk-amd64.wxs [hunk 4]
```diff
@@ -211,6 +211,9 @@
         <File Id="time.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\time.h" Checksum="yes" />
       </Component>
 
+      <Component Id="generic_base.h" Directory="WindowsSDK_usr_include_os" Guid="348df76b-0642-4178-a06e-c79ceab474be">
+        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\dispatch\os\generic_base.h" Checksum="yes" />
```
Comment (MaxDesiatov) [inline platforms/Windows/sdk-amd64.wxs:215]:
```suggestion
        <File Id="generic_base.h" Source="$(var.SDK_ROOT)\usr\lib\swift\os\generic_base.h" Checksum="yes" />
```

**Comment by MaxDesiatov** [comment]
The testing PR on `apple/swift` is green now! 🎉
